docs/user_guide/data_prepare_cat.rst

```diff
@@ -64,11 +66,13 @@ The ``rename``, ``nodata``, ``unit_add`` and ``unit_mult`` options are set per v
 
 The yaml file has an *optional* global **meta** data section:
 
-- **root** (optional): root folder for all the data sources in the yaml file.
+- **roots**: root folders for all the data sources in the yaml file.
```

Comment:
Still optional in the case of cloud data for example. 
I would also explain that they actually link to the same folder and are for cross platform support only. Not like a data source per data source check will be done to be able to store data in different folders.